src/elements/label.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import labelStyles from './label.css'
+
+import { classNames } from 'utils'
+
+const Label = ({
+  children,
+  color,
+  className,
+  tag: Tag = 'mark',
+  ...restProps
+}) => (
+  <Tag
+    className={classNames
+      .use('label', color !== 'default' && color, className)
+      .withModule(labelStyles)}
+    {...restProps}
+  >
+    {children}
+  </Tag>
+)
+
+Label.propTypes = {
+  color: PropTypes.oneOf(['default', 'primary', 'success', 'danger']),
```

Comment:
You forgot to add a tag here. (it's missing in heading too)

![label](https://user-images.githubusercontent.com/12775440/70734920-713fac00-1d0d-11ea-9ce5-352dfb34bf95.PNG)

Reply:
I want to somehow ignore this generic prop globally but there is no ignore list. I know how to do it and will do it later. It's not a priority.

You know that the tag is always a React node.